cmd/root.go

```diff
@@ -49,6 +52,8 @@ var partialCmd = &cobra.Command{
 
 func init() {
 	partialCmd.Flags().BoolVarP(&dryrun, "dry-run", "d", false, "perform dry run without deleting anything")
+	partialCmd.Flags().StringVarP(&channels, "skip-channel", "s", "", "skip message deletion for specified channels")
```

Comment:
What format does the list of channels need to be passed with?

Reply:
They can be passed separated by a space in quotes like "123456 654321 98765"